tests/e2e/identity-flows.test.ts

```diff
@@ -0,0 +1,178 @@
+import {
+  openBrowser,
+  closeBrowser,
+  text,
+  click,
+  screenshot,
+  below,
+  waitFor,
+  write,
+  into,
+  textBox,
+  button,
+  checkBox,
+} from "taiko";
+
+import { authenticateToConnect, getCode, step } from "./utils";
```

Comment:
I create a `index.ts` file in `utils` to avoid having multiple import and pollute top of files.